modules/client/src/Wallet.ts

```diff
@@ -0,0 +1,119 @@
+import { ethers as eth } from 'ethers';
+import Web3 from 'web3';
+import { ConnextClientOptions } from './Connext';
+import { Provider } from './types'
+
+export default class Wallet extends eth.Signer {
```

Comment:
I have a feeling this class (if it's used by others) will need a more specific name, something like `ConnextWallet` or `SigningWallet` or something.

If wallets integrate this package, the nomenclature here may be confusing

Reply:
Also, when we merge this let's make sure to have a correspondingly named PR in the card (and probably docs) that implement these updates

Reply:
But I think it'll be `Connext.Wallet` right? Or `import { Wallet } from 'Connext'`, they can always say `import { Wallet as ConnextWallet } from 'Connext'` if they want to avoid overriding.

Reply:
No no no this `Wallet` class is only used internally, there shouldn't be any reason that the daicard or metamask or anyone needs to access this wallet class directly. It's attached to the `ConnextInternal` instance but I see no reason to expose this on the `ConnextClient` instance that the end user interacts with.